great_expectations/data_context/data_context.py

```diff
@@ -3668,11 +3668,14 @@ def _test_instantiation_of_profiler_from_yaml_config(
             config_defaults={"module_name": "great_expectations.rule_based_profiler"},
         )
 
-        profiler_anonymizer = ProfilerAnonymizer(self.data_context_id)
+        profiler_anonymizer: ProfilerAnonymizer = ProfilerAnonymizer(
+            self.data_context_id
+        )
```

Comment:
As much as I love type annotations, I don't think they're entirely needed when it comes to method calls with clear types (like constructors).

Don't think this hurts at all but perhaps we want to define a standard for type annotations in our style guide.

Reply:
@cdkini Fair enough -- how about leave it like this for now, until we have a standard defined and then fix universally.  My vote is to have them always (comes from `Java` days).

Reply:
better to have them than not at all! LGTM